src/index.js

```diff
@@ -0,0 +1,424 @@
+'use strict';
```

Comment:
This new `src/index.js` looks to be the same as `index.js`. Which file is actually being used? Can we delete one?

Reply:
Correct, we will delete the top-level one before merging. I left it there for ease of diff-checking as moving it into the `src` directory was not picked up by git as a change to the same file